### ISSUE TITLE
[Content organization] Your content or feature isn't placed within an appropriate or relevant area of VA.gov. (02.05.2)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#39476]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA Dashboard

#### VFS feature name
Payment Information

#### Point of Contact/Reviewers
Cheryl Evans (@CherylEvans) - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content or feature isn't placed within an appropriate or relevant area of VA.gov.

### Issue Details
Payment history error should have the heading Benefit Payment and Debts above it for consistency to the other Benefit Payment and Debts pages.  Payment history error warning is at the top of the page by itself, with no relevant heading or anything else that would give context.  

### VA.gov Experience Standard
[Category Number 02, Issue Number 05](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.1

---

### Platform Recommendation
Place the heading Payments and Debts above the Payment history error warning, or use some other means to give this warning message context.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by hdjustice** [comment]
4/6/22 - May be resolved along with current My VA Audit for conditional states - team will address the placement of error messages.

**Comment by aagosto90** [comment]
This has been resolved with the latest iteration of payments where we show errors within the Outstanding Debts and Benefit Payments section and not at the top of the page.
